### ISSUE TITLE
[Error] When inferring llama with multiple GPU (GPU>1)

### ISSUE THREAD

**Issue by marscrazy** [HazyResearch/flash-attention#195]
Thanks for your great work in improving the performance of the transformer-based model.

I have tried to infer the llama model with the code with 4 GPUs (A100).

However, I meet an error. It's very grateful that you can offer some advice.
<img width="877" alt="image" src="https://user-images.githubusercontent.com/4392184/235037041-fb8fe5b9-c1d1-42a4-ae65-7f9bf1bc8d91.png">

**Comment by tridao** [comment]
No, multi-GPU inference for LLaMa is not implemented right now. The inference stuff is just for my own experimentation.

**Comment by marscrazy** [comment]
Thank. I will try to fix this.
